sdk/keyvault/keyvault-keys/src/core/keyVaultClient.ts

```diff
@@ -111,47 +81,23 @@ class KeyVaultClient extends KeyVaultClientContext {
    * @param [options] The optional parameters
    * @returns Promise<Models.ImportKeyResponse>
    */
-  importKey(
-    vaultBaseUrl: string,
-    keyName: string,
-    key: Models.JsonWebKey,
-    options?: Models.KeyVaultClientImportKeyOptionalParams
-  ): Promise<Models.ImportKeyResponse>;
+  importKey(vaultBaseUrl: string, keyName: string, key: Models.JsonWebKey, options?: Models.KeyVaultClientImportKeyOptionalParams): Promise<Models.ImportKeyResponse>;
   /**
    * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
    * @param keyName Name for the imported key.
    * @param key The Json web key
    * @param callback The callback
    */
-  importKey(
-    vaultBaseUrl: string,
-    keyName: string,
-    key: Models.JsonWebKey,
-    callback: coreHttp.ServiceCallback<Models.KeyBundle>
-  ): void;
+  importKey(vaultBaseUrl: string, keyName: string, key: Models.JsonWebKey, callback: coreHttp.ServiceCallback<Models.KeyBundle>): void;
   /**
    * @param vaultBaseUrl The vault name, for example https://myvault.vault.azure.net.
    * @param keyName Name for the imported key.
    * @param key The Json web key
    * @param options The optional parameters
    * @param callback The callback
    */
-  importKey(
-    vaultBaseUrl: string,
-    keyName: string,
-    key: Models.JsonWebKey,
-    options: Models.KeyVaultClientImportKeyOptionalParams,
-    callback: coreHttp.ServiceCallback<Models.KeyBundle>
-  ): void;
-  importKey(
-    vaultBaseUrl: string,
-    keyName: string,
-    key: Models.JsonWebKey,
-    options?:
-      | Models.KeyVaultClientImportKeyOptionalParams
-      | coreHttp.ServiceCallback<Models.KeyBundle>,
-    callback?: coreHttp.ServiceCallback<Models.KeyBundle>
-  ): Promise<Models.ImportKeyResponse> {
+  importKey(vaultBaseUrl: string, keyName: string, key: Models.JsonWebKey, options: Models.KeyVaultClientImportKeyOptionalParams, callback: coreHttp.ServiceCallback<Models.KeyBundle>): void;
```

Comment:
the formatting here seems to be strictly worse, maybe because someone prettier'd it?

Reply:
Yeah I think so. The new changes in my PR were from generator.

Reply:
I think in general we shouldn't touch generated code by ourselves.